superset/assets/src/components/FilterableTable/FilterableTable.jsx

```diff
@@ -173,13 +174,19 @@ export default class FilterableTable extends PureComponent {
   }
 
   getCellContent({ cellData, columnKey }) {
-    const content = String(cellData);
+    let content = String(cellData);
```

Comment:
We generally prefer to use immutable consts over lets. I think this would probably be cleaner if you just handled the `cellData === null` case first before all this other logic. something like:
```
if (cellData === null) {
  return this.complexColumns[columnKey] 
    ? ''
    : (
        <i className="null-cell-render">
          NULL
        </i>
      )
}
```

I'm actually not sure if truncating null to empty string is the desired behavior for complex columns though. @betodealmeida would be a good person to take a look at this too

Reply:
Per your recommendation, I will add the following code at the beginning before the rest of the logic:
```
    if (cellData === null) {
      return (
        <i className="null-cell">
          NULL
        </i>
      );
    }
```